tests/e2e/docusaurus2-graphql-doc-generator-multi-instance-1.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  id: "config-A",
+  schema: "data/tweet-A.graphql",
+  rootPath: "./docs-tweet-A",
```

Comment:
I'd rather limit the number of local schemas to maintain
```suggestion
  id: "tweet",
  schema: "data/tweet.graphql",
  rootPath: "./doc-tweet",
```

Reply:
Sure. Addressed in 9b550e2.